code/game/objects/structures/mirror.dm

```diff
@@ -19,7 +19,7 @@
 			to_chat(user, "<span class='notice'>Your reflection appears distorted on the surface of \the [src].</span>")
 
 	if(ishuman(user))
-		var/datum/nano_module/appearance_changer/AC = ui_users[user]
+		var/datum/vueui_module/appearance_changer/AC = ui_users[user]
```

Comment:
This looks like oldcode.

Please use the "current" way of interacting with UIs (`ui_interact` proc with `SSvueui-get_open_ui()`)